src/app/resources.py

```diff
@@ -74,26 +76,27 @@ def delete(self, *args, **kwargs):
         raise NotImplementedError
 
 
+@api.representation('/json')
 class UserListResource(Resource):
     def get(self):
         try:
             users = session.query(User).all()
         except SQLAlchemyError:
-            return {'msg': SQLAlchemyError}, 500,  # Internal Server Error
-        return {'users': UserSchema(many=True).dump(users)}, 200,  # OK
```

Comment:
Are you sure that a good practice to return objects instead of json?

Reply:
Response generates in output_json and that function return response with data, code and headers. RespresentationApi class determinate which response format we are support. I guess this is the solution of issue #56 
https://flask-restful.readthedocs.io/en/latest/extending.html#response-formats